pyproject.toml

```diff
@@ -1,3 +1,12 @@
+[build-system]
+# Minimum requirements for the build system to execute.
+requires = [
+    "setuptools",
+    "wheel",
+    "Cython>=0.28.5",
```

Comment:
Do we need cython? I don't think that we use any cython, so I don't think that we need it here even for the build (unlike scikit-learn).